servarr/config/qbittorrent/init-qBitTorrent.py

```diff
@@ -0,0 +1,164 @@
+#!/usr/local/bin/python3
+
+import os
+import sys
+import logging
+import base64
+import hashlib
+import jinja2
+
+MIN_PASS_LEN = 8
+QBITTORRENT_CONF_FILENAME = "qBittorrent.conf"
+QBITTORRENT_CONF_FILEPATH = "/mnt/qBittorrent/config"
+QBITTORRENT_CONF_TEMPLATE = r"""
+[Application]
+FileLogger\Age=1
+FileLogger\AgeType=1
+FileLogger\Backup=true
+FileLogger\DeleteOld=true
+FileLogger\Enabled=true
+FileLogger\MaxSizeBytes=66560
+FileLogger\Path=/config/qBittorrent/logs
+
+[BitTorrent]
+Session\DefaultSavePath=/downloads
+Session\ExcludedFileNames=
+Session\Port=50413
+Session\QueueingSystemEnabled=true
+Session\TempPath=/downloads/temp
+
+[Core]
+AutoDeleteAddedTorrentFile=Never
+
+[LegalNotice]
+Accepted=true
+
+[Meta]
+MigrationVersion=6
+
+[Network]
+Proxy\HostnameLookupEnabled=false
+Proxy\Profiles\BitTorrent=true
+Proxy\Profiles\Misc=true
+Proxy\Profiles\RSS=true
+
+[Preferences]
+General\Locale=en
+MailNotification\req_auth=true
+WebUI\AuthSubnetWhitelist=192.168.0.0/24, 10.1.0.0/24
```

Comment:
Are those subnets valid for everyone or only for your env?

Reply:
These are the default subnets for k8s pods/services.. of course users can use other means.
Anyway, this is not necessary. I will remove it 👍🏻